ext/ftp/tests/ftp_ssl_connect_error.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test ftp_ssl_connect() function : error conditions
+--SKIPIF--
+<?php
+$ssl = 1;
+require 'skipif.inc';
+if (!function_exists("ftp_ssl_connect")) die("skip ftp_ssl is disabled");
+?>
+--FILE--
+<?php
+echo "*** Testing ftp_ssl_connect() function : error conditions ***\n";
+echo "\n-- Testing ftp_ssl_connect() function on failure --\n";
+var_dump(ftp_ssl_connect('127.0.0.1'));
+
+echo "\n-- Testing ftp_ssl_connect() function timeout warning for value 0 --\n";
+ftp_ssl_connect('127.0.0.1', 21, 0);
+
+?>
```

Comment:
@chancegarcia idd, the zend_parse_parameters() function translates the parameters sent to a function call into C variables. You take a look at the README.PARAMETER-PARSING-API or something similar in the root of php-src that has more in-depth about how this works and the meanings =)